release notes/v0.44.0.md

```diff
@@ -0,0 +1,227 @@
+}
+
+function stringToArrayBuffer(str) {
+  var buf = new ArrayBuffer(str.length * 2); // 2 bytes for each char
+  var bufView = new Uint16Array(buf);
+  for (var i = 0, strLen = str.length; i < strLen; i++) {
+    bufView[i] = str.charCodeAt(i);
+  }
+  return buf;
+}
+```
+</details>
+
+The list of supported APIs and algorithms you could find in the project's [README](https://github.com/grafana/xk6-webcrypto/blob/v0.1.0/README.md#current-state)
+
+### Add sampling capabilities to the experimental tracing module [#2886](https://github.com/grafana/k6/pull/2886)
+
+This release adds sampling capabilities to the tracing module. You can now specify a sampling rate with `sampling` option when initializing a Client or in `tracing.InstrumentHTTP` function.
+
+### `browserContext.addCookies` [xk6-browser#760](https://github.com/grafana/xk6-browser/pull/760)
+
+Websites require cookies to help store session details which will be used by the backend to retrieve the state. For example, when a user logins into a website then the backend will store login session details in a cookie, which can be used in future actions performed on the website. For the given example, to speed up the test run, instead of having to first go through the login flow, it would be quicker to set a known valid cookie so that the user can easily test a different feature of the website. With this new feature cookies can be added to a `browserContext` and all new `page`s created from this `browserContext` will have the cookie assigned to them. Thanks @zucchinho for implementing the feature!
+
+```javascript
+const context = browser.newContext()
+context.addCookies([{name: 'myCookie', value: 'hello world', url: 'https://test.k6.io'}])
+const page = context.newPage()
+```
+
+### `browserType.Connect` [xk6-browser#800](https://github.com/grafana/xk6-browser/pull/800)
+
+There are cases where the user may want to connect to a remote browser instance where they have more control over the browser lifecycle, such as when working in a resource bound environment. This feature enables users to connect to a manually started Chrome/Chromium browser instance. It's a simple case of replacing `browser.launch` with `browser.connect` and supplying the CDP url as the first argument. Not all `launch` options will work with `connect` since the browser instance should already have started prior to working with `connect`. Since we assume that the user had decided to take ownership of starting the browser, we have made `browser.close` a NOOP when working with `browser.connect`, so the user will need to close the browser themselves.
+
+```javascript
+const browser = chromium.connect('ws://127.0.0.1:1234/devtools/browser/e3bb7e53-ad0f-46f3-ae89-a8416868f4ce')
+const page = browser.newPage();
+```
+
+### Web Vitals are now natively supported by the xk6-browser module [xk6-browser#836](https://github.com/grafana/xk6-browser/pull/836) [xk6-browser#847](https://github.com/grafana/xk6-browser/pull/847)
```

Comment:
We have an open [PR](https://github.com/grafana/k6-docs/pull/1131) to update the k6 docs to discuss web vitals in more detail. Do you think we should reference this here too? See https://mdr-ci.staging.k6.io/docs/refs/pull/1131/merge/using-k6-browser/browser-metrics/

We probably have to merge the docs changes first.

Reply:
I think that's a good idea. Do you know what the final url will be? The only issue i see is if this page is moved in the future.

Reply:
The URL will be similar https://k6.io/docs/using-k6-browser/browser-metrics/, but the content is currently different.

Reply:
@mdcruz when is this expected to be released?

Reply:
@mstoykov It is ready to be merged :) So I can release it at the same time, maybe? Or before?

Reply:
As far as I remember, an order of things is that we should first do the "archive" of the docs first (but we still do that at the last moment), then merge docs' PRs for the new functionality and do the docs release, we could be still before issuing the k6's release.

Reply:
So we can change the url to the new one and will be correct in a little while :P

Reply:
I've updated it, please see this commit: 9d0e54ad409613e830b59414a2580495512d3810